sentry.client.config.ts

```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://a1dc8cd72a62aacab0dd520eee207398@o4505687617175552.ingest.sentry.io/4505687622090752",
+
+  integrations: [new Sentry.BrowserTracing()],
+
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1,
+
+  // Setting this option to true will print useful information to the console while you're setting up Sentry.
+  debug: false,
+
+  environment: "production",
+
+  tracePropagationTargets: [/^https:\/\/beta\.yuanjian\.org/],
```

Comment:
can you use /.*\.yuantu\..*/ and /.*\.yuanjian\..*/? what's the downside if we set the target too widely, e.g. /.*/ ?

Reply:
The default value was `['localhost', /^\//]`, I was thinking to limit specifically to `beta.yuanjian.org` to filter unnecessary errors that happened during development. `/.*/` will be used for the latest update